test/common/common/utility_test.cc

```diff
@@ -58,6 +58,8 @@ TEST(InputConstMemoryStream, All) {
   }
 }
 
+TEST(StringUtil, WhitespaceChars) { EXPECT_EQ(" \t\f\v\n\r", StringUtil::WhitespaceChars); }
```

Comment:
I'd instead write a 5-line test: one line for each character you want to test.
  EXPECT_NE(nullptr, strchr(StringUtil::WhitespaceChars, ' '));
  EXPECT_NE(nullptr, strchr(StringUtil::WhitespaceChars, '\t'));
this test wouldn't be dependent on the order of the whitespace characters, which is not important.

Reply:
Good call!